doc/api/esm.md

```diff
@@ -687,7 +689,37 @@ executed under:
 node --experimental-wasm-modules index.mjs
 ```
 
-would provide the exports interface for the instantiation of `module.wasm`.
+would provide the exports interface for the instantiation of `library.wasm`.
+
+### Wasm Source Phase Imports
+
+<!-- YAML
+added: REPLACEME
+-->
+
+The [Source Phase Imports][] proposal allows the `import source` keyword
+combination to import a `WebAssembly.Module` object directly, instead of getting
+a module instance already instantiated with its dependencies.
+
+This is useful when needing custom instantiations for Wasm, while still
+resolving and loading it through the ES module integration.
+
+For example, to create multiple instances of a module, or to pass custom imports
+into a new instance of `library.wasm`:
+
+<!-- eslint-skip -->
```

Comment:
Could we instead add the required plugin? Can happen in a follow-up PR

Reply:
Can you rebase and enable linter now that support for linting that syntax has landed on main
```suggestion
```

Reply:
Added, thanks.